Snippets/Sqs/Sqs_6/Usage.cs

```diff
@@ -0,0 +1,391 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.S3;
+using Amazon.SimpleNotificationService;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using NServiceBus;
+using NServiceBus.Pipeline;
+
+
+class Usage
+{
+    void AssumePermissionsOnPolicy(EndpointConfiguration config)
+    {
+        #region assume-permissions
+
+        var transport = new SqsTransport();
+        
+        transport.Policies.SetupTopicPoliciesWhenSubscribing = false;
```

Comment:
does that not work with object initializers?

Reply:
I don't think so 😞

Reply:
Nope :(